app/controllers/locale_ninja/dashboard_controller.rb

```diff
@@ -5,12 +5,12 @@ class DashboardController < ApplicationController
     before_action :set_client, only: [:index]
 
     def index
-      @locales_count = LocaleHelper.locales_count(@client)
+      @locales_count = LocaleHelper.locales_count(@client, @client.default_branch)
       @repo = @client.repo_information
-      translation_branches = @client.branches.filter { |branch| branch[:name].ends_with?('__translations') }
+      translation_branches = @client.branch_names.filter { |branch_name| branch_name.ends_with?(GithubApiService::TRANSLATIONS_SUFFIX) }
```

Comment:
we should probably do this filter in the github API service as we already have a function that takes non_translation branches